File: C++/M01_ProgrammingBasics/L04_ForLoop/Lab/Solutions/P09_LeftAndRightSum.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {

	int n;
	cin >> n;

	int leftSide = 0;
	for (int i = 0; i < n; i++) {
		int num;
		cin >> num;

		leftSide += num;
	}

	int rightSide = 0;
	for (int i = 0; i < n; i++) {
		int num;
		cin >> num;

		rightSide += num;
	}

	if (leftSide == rightSide) {
		cout << "Yes, sum = " << leftSide << endl;
	} else {
		cout << "No, diff = " << abs(rightSide - leftSide) << endl;
	}

	return 0;
}

File: Python/M01_ProgrammingBasics/L04_ForLoop/Lab/Solutions/P09LeftAndRightSum.py
num = int(input())
left = 0
right = 0

for each in range(num):
    left += int(input())

for each in range(num):
    right += int(input())

diff = abs(left - right)

if left == right:
    print(f"Yes, sum = {right}")
else:
    print(f"No, diff = {diff}")